app/assets/stylesheets/general/_titles.scss

```diff
@@ -49,7 +49,21 @@ h6 {
 
 h1, h2, h3, h4, h5, h6 {
   .header_anchor {
-    text-decoration: none;
+    float: left;
```

Comment:
We shouldn't use floats as they tend to break layouts. `position: absolute` might be more suitable here.

Reply:
I took this from the way GitHub was doing it. `position: absolute` means the headers need `position: relative`, which is something I wanted to avoid. `float: left` seems to work for allowing `width` and `height` to change.

Reply:
@netux Any particular reason you don't want to use `position: absolute`?

Reply:
> `position: absolute` means the headers need `position: relative`

I'd argue that changes the layout even more than `float`. And I see no reason to change it when this works fine too.

Any particular examples of `float` breaking layout you may like to point out? Again, I'm mostly copying what GitHub does so I assume they didn't find any issues with it.

Reply:
`position: relative` doesn't change anything about the layout outside of its children.

From [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/position#syntax):
> `relative`
>
>    The element is positioned according to the normal flow of the document, and then offset relative to itself based on the values of top, right, bottom, and left. The offset does not affect the position of any other elements; thus, the space given for the element in the page layout is the same as if position were static.

Since headings are block-level elements anyways, nothing outside of the heading's own context would be affected.